openstack-manager-blueprint.yaml

```diff
@@ -687,7 +687,7 @@ node_templates:
       resource_id: { get_input: management_subnet_name }
       subnet:
         ip_version: 4
-        cidr: 10.67.79.0/24
+        cidr: 10.67.0.0/16
```

Comment:
Well, it seems kind of reasonable, but aren't addresses 128.x.x.x from the public range?